_includes/markdown/JavaScript.md

```diff
@@ -2,96 +2,55 @@
 
 ### Cache DOM Selections
 
-It's a common JavaScript mistake to reselect something unnecessarily. For example, every time a menu button is clicked, we do not need to reselect the menu. Rather, we select the menu once and cache its selector. This applies whether you are using jQuery or not. For example:
+It's a common JavaScript mistake to reselect something unnecessarily. For example, every time a menu button is clicked, we do not need to reselect the menu. Rather, we select the menu once and cache its selector. This applies whether you are using a library or not. For example:
 
-non-jQuery Uncached:
+Uncached:
 
 ```javascript
-var hideButton = document.getElementsByClassName( 'hide-button' )[0];
-hideButton.onclick = function() {
-    var menu = document.getElementById( 'menu' );
+const hideButton = document.querySelector( '.hide-button' );
+
+hideButton.addEventListener( 'click', () => {
+    const menu = document.getElementById( 'menu' );
     menu.style.display = 'none';
-}
+} );
 ```
 
-non-jQuery Cached:
+Cached:
 
 ```javascript
-var menu = document.getElementById( 'menu' );
-var hideButton = document.getElementsByClassName( 'hide-button' )[0];
-hideButton.onclick = function() {
+const menu = document.getElementById( 'menu' );
+const hideButton = document.querySelector( '.hide-button' );
+
+hideButton.addEventListener( 'click', () => {
     menu.style.display = 'none';
 }
 ```
 
-jQuery Uncached:
-
-```javascript
-var $hideButton = jQuery( '.hide-button' );
-$hideButton.on( 'click', function() {
-    var $menu = jQuery( '#menu' );
-    $menu.hide();
-});
-```
-
-jQuery Cached:
-
-```javascript
-var $menu = jQuery( '#menu' );
-var $hideButton = jQuery( '.hide-button' );
-$hideButton.on( 'click', function() {
-	$menu.hide();
-});
-```
-Notice how in cached versions we are pulling the menu selection out of the event handler so it only happens once. Non-jQuery cached is not surprisingly the [fastest way to handle this situation](https://jsperf.com/dom-selection-caching).
+Notice how, in cached versions, we are pulling the menu selection out of the event listener so it only happens once. The cached version is, not surprisingly, the [fastest way to handle this situation](https://jsperf.com/dom-selection-caching).
 
 #### Event Delegation
 
 Event delegation is the act of adding one event listener to a parent node to listen for events bubbling up from its children. This is much more performant than adding one event listener for each child element. Here is an example:
 
-Without jQuery:
-
 ```javascript
-document.getElementById( 'menu' ).addEventListener( 'click', function( event ) {
-    var currentTarget = event.currentTarget;
-    var target = event.target;
+document.getElementById( 'menu' ).addEventListener( 'click', ( e ) => {
+
+    const currentTarget = e.currentTarget;
+    let target = event.target;
 
     if ( currentTarget && target ) {
       if ( 'LI' === target.nodeName ) {
```

Comment:
All of these changes look great. If you review this code, I really think it could be updated to leverage `closest()`. See: https://developer.mozilla.org/en-US/docs/Web/API/Element/closest